examples/most-redis-docker/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.DS_Store
```

Comment:
Is this file redundant now?  Should I remove it?

Reply:
Good catch.  Yeah, I think it can be removed.